### ISSUE TITLE
Can't upgrade to victory@0.21.0 deps: Curve.renderLine throws

### ISSUE THREAD

**Issue by chrisbolin** [FormidableLabs/victory-native#101]
`victory-native` should stay up-to-date with `victory`. The last victory, `victory@0.21.0` depends on the following versions:

```json
"victory-chart": "^21.0.0",
"victory-core": "^16.0.0",
"victory-pie": "^11.1.2"
```

However, if we upgrade `victory-native` to those deps, we get the following error on the demo:

```sh
ExceptionsManager.js:63 paths.map is not a function
lib/components/victory-primitives/curve.js:15:17
```

which is

```jsx
  renderLine(paths, style, events) {
    const { role, shapeRendering, className } = this.props;
    const nativeStyle = NativeHelpers.getStyle(style);
    return paths.map((path, index) => {
      return (
        <Path
          key={`area-stroke-${index}`}
          shapeRendering={shapeRendering || "auto"}
          role={role || "presentation"}
          d={path}
          className={className}
          {...nativeStyle}
          {...events}
        />
      );
    });
  }
```

CC @boygirl

**Comment by boygirl** [comment]
@chrisbolin this is expected. I introduced many breaking changes for native in the last release. I believe most of them are listed in the [changelog](https://github.com/FormidableLabs/victory/blob/master/CHANGELOG.md), but I may have missed some, especially in the primitive components. 

There will also be other changes necessary for native in order to support polar coordinate charts:
- native version of `VictoryPolarAxis`
- native version of the new `Arc` primitive
- `defaultPolarAxes` prop default for the native version of `VictoryChart`
- changes to the containers paralleling what changes in the main library.

**Comment by chrisbolin** [comment]
Perfect! Thanks, Lauren 👍

**Comment by chrisbolin** [comment]
Fixed in `0.11.1`
